src/gridstack-dd.ts

```diff
@@ -9,7 +9,7 @@
 
 import { GridStackDDI } from './gridstack-ddi';
 import { GridItemHTMLElement, GridStackNode, GridStackElement, DDUIData, DDDragInOpt } from './types';
-import { GridStack } from './gridstack';
+import { GridStack, GridStackEventHandlerCallback } from './gridstack';

```

Comment:
you have it defined on type, not gridstack per say....

Reply:
this should be from types, my VScode automatically imported it from the wrong place. Will keep note 👍

Reply:
yeah VS code that sometimes...